libs/wingsdk/src/std/array.ts

```diff
@@ -2,14 +2,22 @@
 // They should not be consumed directly by users.
 // TODO: These should be interfaces, currently Wing does not support interface JSII imports
 
-import { T1 } from "./util";
+import { T1 } from "./generics";
+import { Code, InflightClient } from "../core";
 
 /**
  * Immutable Array
  *
  * @typeparam T1
  */
 export class ImmutableArray {
+  /**
+   * @internal
+   */
+  public static _toInflightType(): Code {
+    return InflightClient.forType(__filename, "ImmutableArray");
```

Comment:
Looks like the 2nd arg here (`clientClass`) is always `this.name`. Is the extra arg here for extra flexibility?

Reply:
TIL- DONE!